templates/demo-store/app/routes/($lang).cart.tsx

```diff
@@ -18,9 +18,10 @@ import type {
 } from '@shopify/hydrogen/storefront-api-types';
 import {isLocalPath, getCartId} from '~/lib/utils';
 import {CartAction, type CartActions} from '~/lib/type';
+import {CartFormInput, CartFormInputAction} from '@shopify/hydrogen';
 
 export async function action({request, context}: ActionArgs) {
-  const {session, storefront} = context;
+  const {session, storefront, cart} = context;
   const headers = new Headers();
   let cartId = getCartId(request);
```

Comment:
This one feels off to me coming from `libs`? - Is this temporary? 

My expectation `getCartId` to be another cart method accessed via the context.